docs/tutorials/HD143006_Part_1.py

```diff
@@ -0,0 +1,201 @@
+    DEC[-1] + CDELT2 / 2,
+)  # [arcsec]
+
+# plotting the clean fits
+plt.imshow(np.squeeze(clean_fits), origin='lower', extent=ext)
+# limiting the x and y axes so that we only plot the disk
+plt.xlim(left=0.75, right=-0.75)
+plt.ylim(top=0.75, bottom=-0.75)
+# axis labels
+plt.xlabel(r"$\Delta \alpha \cos \delta$ [${}^{\prime\prime}$]")
+plt.ylabel(r"$\Delta \delta$ [${}^{\prime\prime}$]")
+
+# This is the image produced by the CLEAN algorithm used by the DSHARP team.  In Part 1 of this tutorial we will be creating an MPoL Gridder object and  the diagnostic dirty image. [**Part 2**](https://mpol-dev.github.io/MPoL/tutorials/HD143006_Part_2.html) of the tutorial will cover the optimization loop of the model to create the RML image. 
+#
+# ### Plotting the Visibilities
+#
+# To create the dirty image, we will use the extracted visibilities from the npz file and the MPoL Gridder and Coordinates packages.
+#
+
+# load extracted visibilities from npz file
+dnpz = np.load(fname_EV) 
+uu = dnpz['uu']
+vv = dnpz['vv']
+weight = dnpz['weight']
+data = dnpz['data']
+
+# Let's quickly plot the U,V visibilities as seen in the [Cross Validation Tutorial](https://mpol-dev.github.io/MPoL/tutorials/crossvalidation.html) and the [Visread docs](https://mpol-dev.github.io/visread/tutorials/introduction_to_casatools.html#Get-the-baselines).
+
+fig, ax = plt.subplots(nrows=1)
+ax.scatter(uu,vv, s=.5, rasterized=True, linewidths=0.0, c='k')
+ax.scatter(-uu,-vv, s=.5, rasterized=True, linewidths=0.0, c='k')
+ax.set_xlabel(r"$u$ [k$\lambda$]")
+ax.set_ylabel(r"$v$ [k$\lambda$]")
+ax.set_title(r'$U$, $V$ Visibilities')
+
+# As you can see, there are very few visibilities > 7,000 ($k\lambda$), and a very dense region of visibilities between -2000 and 2000 ($k\lambda$). This indicates outliers at the higher frequencies while the bulk of our data stems from these lower frequencies.
+
+# ### Creating the Gridder
+#
+# To create the MPoL Gridder object, we need a `cell_size` and the number of pixels in the width of our image, `npix`. You can read more about these properties in the [GridCoords](https://mpol-dev.github.io/MPoL/api.html#mpol.coordinates.GridCoords) API Documentation.  In the fits header we see our image is 3000x3000 pixels, so `npix=3000`. Getting our cell size in terms of arcseconds is a bit more tricky. Our fits image has a header called `CDELT1` which is the scaling in degrees. To get this into arcseconds we multiply by 3600. We save this as `cdelt_scaling`. `cdelt_scaling` can be negative, and cell_size must be positive so we will take the absolute value of this.
+
+# opening the fits file
+dfits = fits.open(fname_F)
+# printing out the header info
+dfits.info()
+
+cdelt_scaling = dfits[0].header['CDELT1'] * 3600 # scaling [arcsec]
+cell_size = abs(cdelt_scaling) # [arcsec]
+# closing fits file
+dfits.close()
+
+from mpol import gridding, coordinates
+
+# creating Gridder object
+coords = coordinates.GridCoords(cell_size=cell_size, npix=3000)
```

Comment:
Should i change this to 512? I thought it should be the same size as the fits image dimensions but this creates a large memory need.

Reply:
3000 is indeed too large---the DSHARP FITS files are very large just to fill the full primary beam. We don't need to do that for the purposes of the tutorial. So long as you have enough pixels to comfortably fit the image. 512 might be enough, or maybe you'll need to go to 800 or 1024.

Reply:
Sounds good, I will try a few out